src/app/core/services/in-memory-storage.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { IStorage } from './i-storage.interface';
+
+@Injectable()
+export class InMemoryStorageService implements IStorage {
```

Comment:
I decided to go for InMemoryStorageService cause I wanted this storage to be cleared on page refresh. I hide it behind interface to allow to change the behaviour to use browser's LocalStorage if the need comes.

Reply:
Hymm, as far as I know, Kamil P i using it class to it would be good to notify him about this change